tidy.Rmd

```diff
@@ -1,564 +1,534 @@
 
-> "Tidy datasets are all alike, but every messy dataset is messy in its
-> own way." --– Hadley Wickham
+> "Todos los datos ordenados se parecen unos a otros, pero cada dato desordenado lo es a su manera" --- Hadley Wickham
 
-In this chapter, you will learn a consistent way to organise your data in R, an organisation called __tidy data__. Getting your data into this format requires some upfront work, but that work pays off in the long term. Once you have tidy data and the tidy tools provided by packages in the tidyverse, you will spend much less time munging data from one representation to another, allowing you to spend more time on the analytic questions at hand.
+En este capítulo aprenderás una metodología consistente para organizar datos en R, a esta metodología le llamaremos __tidy data__ (datos ordenados). Llevar tus datos a este formato requiere algo de trabajo previo, sin embargo dicho trabajo tiene retorno positivo en el largo plazo. Una vez que tengas tus datos ordenados y las herramientas para ordenar datos que provee el tidyverse, vas a gastar mucho menos tiempo pasando de una forma de representar datos a otra, permietiéndote destinar más tiempo a las preguntas analíticas.
 
-This chapter will give you a practical introduction to tidy data and the accompanying tools in the __tidyr__ package. If you'd like to learn more about the underlying theory, you might enjoy the *Tidy Data* paper published in the Journal of Statistical Software, <http://www.jstatsoft.org/v59/i10/paper>.
+Este capítulo te dará una introducción práctica a tidy data y las herramientas que provee el paquete __tidyr__. Si desear saber más acerca de la teoría subyacente, puede que te guste el artículo *Tidy Data* publicado en la revista Journal of Statistical Software, <http://www.jstatsoft.org/v59/i10/paper>.
 
-### Prerequisites
+### Prerequisitos
 
-In this chapter we'll focus on tidyr, a package that provides a bunch of tools to help tidy up your messy datasets. tidyr is a member of the core tidyverse.
+En este capítulo nos enfocaremos en tidyr, un paquete que provee un conjunto de herramientas que te ayudarán a ordenar datos desordenados. tidyr es parte del núcleo del tidyverse.
 
 ```{r setup, message = FALSE}
 library(tidyverse)
+library(datos)
 ```
 
-## Tidy data
+## Datos ordenados
 
-You can represent the same underlying data in multiple ways. The example below shows the same data organised in four different ways. Each dataset shows the same values of four variables *country*, *year*, *population*, and *cases*, but each dataset organises the values in a different way.
+Puedes representar la misma información de múltiples formas. El ejemplo a continuación muestra los mismos datos ordenados de cuatro manera distintas. Cada dataset muestra los mismos valores de cuatro variables *pais*, *anio*, *poblacion* y *casos*, pero cada dataset organiza los valores de forma distinta.
 
 ```{r}
-table1
-table2
-table3
+tabla1
```

Comment:
Este comentario puede que no tenga importancia, pero veo que los tableros de `datos` tienen variables de factores, lo cual no es el caso en los datos de `tidyr`:

``` r
library(datos)
library(tidyr)
#> Warning: package 'tidyr' was built under R version 3.5.2

tidyr::table4a
#> # A tibble: 3 x 3
#>   country     `1999` `2000`
#> * <chr>        <int>  <int>
#> 1 Afghanistan    745   2666
#> 2 Brazil       37737  80488
#> 3 China       212258 213766
datos::tabla4a
#> # A tibble: 3 x 3
#>   pais       `1999` `2000`
#>   <fct>       <int>  <int>
#> 1 Afganistan    745   2666
#> 2 Brasil      37737  80488
#> 3 China      212258 213766
```

Created on 2019-03-07 by the [reprex
package](http://reprex.tidyverse.org) (v0.2.0).

Los factores no afectan para nada las operaciones de este capítulo, pero no sé si ese pueda ser el caso en otros capítulos. Lo dejo aquí más como un FYI, pero no como sugerencia de corrección.